README.rst

```diff
@@ -27,12 +27,16 @@ Install Log Anomaly Detector (LAD):
 
 .. code-block:: shell
 
-   pip install git+https://github.com/AICoE/log-anomaly-detector.git
+   pip install log-anomaly-detector
 
 .. note::
 
    LAD requires python 3.6
```

Comment:
@zmhassan I think the formatting didn't work here but it worked in the readthedocs page